integration-cli/docker_cli_events_test.go

```diff
@@ -640,3 +640,15 @@ func (s *DockerRegistrySuite) TestEventsImageFilterPush(c *check.C) {
 	out, _ = dockerCmd(c, "events", "--since=0", "-f", "image="+repoName, "-f", "event=push", "--until="+strconv.Itoa(int(since)))
 	c.Assert(out, checker.Contains, repoName+": push\n", check.Commentf("Missing 'push' log event"))
 }
+
+// #18453
+func (s *DockerRegistrySuite) TestEventsContainerFilter(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	out, _ := dockerCmd(c, "run", "--name=foo", "-d", "busybox", "top")
+	c1 := strings.TrimSpace(out)
+	out, _ = dockerCmd(c, "run", "--name=bar", "-d", "busybox", "top")
+	c2 := strings.TrimSpace(out)
+	out, _ = dockerCmd(c, "events", "-f", "container=foo")
+	c.Assert(out, checker.Contains, c1, check.Commentf("Missing event of container (foo)"))
+	c.Assert(out, checker.Contains, c2, check.Commentf("Should not contain event of container (bar)"))
```

Comment:
Ehm, is this correct, because the assert checks if the container **is** in there; should these the opposite, right?